modules/clickhouse/src/main/java/org/testcontainers/containers/ClickHouseInit.java

```diff
@@ -0,0 +1,40 @@
+package org.testcontainers.containers;
+
+import org.testcontainers.containers.wait.strategy.HttpWaitStrategy;
+import org.testcontainers.utility.DockerImageName;
+
+import java.time.Duration;
+
+public class ClickHouseInit {
+    public static final String JDBC_NAME_YANDEX = "clickhouse";
+    public static final String JDBC_NAME_MYSQL = "mysql";
+
+    public static final DockerImageName DEFAULT_IMAGE_NAME = DockerImageName.parse("yandex/clickhouse-server");
+
+    public static final String CLICKHOUSE_DRIVER_CLASS_NAME = "ru.yandex.clickhouse.ClickHouseDriver";
+    public static final String MYSQL_DRIVER_CLASS_NAME = "com.mysql.cj.jdbc.Driver";
+
+    public static final Integer HTTP_PORT = 8123;
+    public static final Integer NATIVE_PORT = 9000;
+    public static final Integer MYSQL_PORT = 9004;
+
+    public static final String databaseName = "default";
+    public static final String username = "default";
+    public static final String password = "";
+
+    static final String TEST_QUERY = "SELECT 1";
+
+    public static final String DEFAULT_TAG = "21.3.8.76";
+
+    public static void Init(GenericContainer<?> container) {
+        container.withExposedPorts(HTTP_PORT, NATIVE_PORT, MYSQL_PORT);
+
+        container.waitingFor(
+            new HttpWaitStrategy()
+                .forStatusCode(200)
+                .forPort(HTTP_PORT)
+                .forResponsePredicate(responseBody -> "Ok.".equals(responseBody))
```

Comment:
Is this response important if we already get 200 status code?

Reply:
I prefer to be strict with these (bad experiences with APIs returning 200 but with a failure message).
In our own driver we check for the "Ok." reply since it is explicitly mentioned in the clickhouse docs as the return value that indicates success.

Reply:
I am not sure I share the preference :) It seems to be a free text value, not even some structured response (e.g. JSON).

I would rely on ClickHouse devs being the good folks who understand the meaning of 200 and would never return 200 in case of an error :)